### PR TITLE
Updated Spanish translation

### PR DESCRIPTION
**Pull request by ignotus666**

### CODE REVIEW

File: src/res/translation/translation_es_ES.ts [hunk 1]
```diff
@@ -1987,7 +1987,7 @@
     <message>
         <location filename="../../clientsettingsdlgbase.ui" line="624"/>
         <source>Local Jitter Buffer</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Jitter Buffer Local</translation>
```
Comment (ann0see) [inline src/res/translation/translation_es_ES.ts:1990]: I think you have to remove the unfinished attribute here too

**Review by ann0see — CHANGES_REQUESTED** [review]
Thank you very much. See my comment

**Comment by ann0see** [comment]
Not sure if you already know that you can translate the new Windows installer too: https://github.com/jamulussoftware/jamulus/pull/1124/files

**Comment by ann0see** [comment]
I'll just post the official checklist here:

I think everything should be ok, but anybody who wants can check it.

- [ ] Translator listed in the `src/util.cpp` **optionally add link to PR or code**
- [ ] Looks consistent
- [ ] Passes `tools/check-wininstaller-translations.sh`

File: src/res/translation/wininstaller/installerlng.nsi [hunk 2]
```diff
@@ -19,3 +19,6 @@
 
 !insertmacro MUI_LANGUAGE "French"
 !include "${ROOT_PATH}\src\res\translation\wininstaller\fr.nsi"
+
+!insertmacro MUI_LANGUAGE "Spanish"
+!include "${ROOT_PATH}\src\res\translation\wininstaller\sp.nsi"
```
Comment (ann0see) [inline src/res/translation/wininstaller/installerlng.nsi:24]: Just a small question here: why did you choose sp.nsi not es.nsi?

File: src/res/translation/wininstaller/sp.nsi [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+     "Para utilizar Jamulus, necesitas un driver ASIO"
+
+LangString ASIO_DRIVER_EXPLAIN ${LANG_SPANISH} \
+     "Jamulus necesita un driver ASIO para proporcionar una baja latencia. Más información:"
+
+LangString ASIO_DRIVER_MORE_INFO ${LANG_SPANISH} \
+     "Más información sobre ASIO en jamulus.io"
+
+LangString ASIO_DRIVER_MORE_INFO_URL ${LANG_SPANISH} \
+     "https://jamulus.io/wiki/Installation-for-Windows#asio"
```
Comment (ann0see) [inline src/res/translation/wininstaller/sp.nsi:34]: I think, you can even link the spanish site here.

**Review by ann0see — APPROVED** [review]
Seems to be good to me. I'll go through the checklist soon. Sorry for the late review.

File: src/res/translation/wininstaller/installerlng.nsi (hunk 2 above)
Comment (ignotus666) [reply src/res/translation/wininstaller/installerlng.nsi]: Good question... Changed.

File: src/res/translation/wininstaller/sp.nsi (hunk 3 above)
Comment (ignotus666) [reply src/res/translation/wininstaller/sp.nsi]: Done.

Comment (ann0see) [reply src/res/translation/wininstaller/sp.nsi]: Great! Thanks. You can test the windows installer if you click on the checks page and download the artifact once the CI finished and do some tests.

File: src/res/translation/wininstaller/installerlng.nsi (hunk 2 above)
Comment (ann0see) [reply src/res/translation/wininstaller/installerlng.nsi]: Great!

**Review by softins — APPROVED** [review]

**Review by hoffie — APPROVED** [review]
Thanks for your work!

Adding the non-installer checklist as well.

- [x] Translator listed in the `src/util.cpp` **optionally add link to PR or code**
- [x] Punctuation and spacing consistent
- [x] Signal words consistent ("ASIO", "Buffer")
- [x] No untranslated strings (`grep unfinished -5 src/res/translation/translation_$TRANSLTION*.ts`)
- [x] Only a single `.ts` file checked in (+ Installer translations)

Not merging yet to give @ignotus666 a chance to check the installer artifact maybe?

File: src/res/translation/wininstaller/sp.nsi (hunk 3 above)
Comment (ignotus666) [reply src/res/translation/wininstaller/sp.nsi]: I don't have Windows but the installer works with WINE - looks ok.

Comment (ann0see) [reply src/res/translation/wininstaller/sp.nsi]: Ok. Great thank you.

**Comment by ann0see** [comment]
@ignotus666 you might need to force push on your Repo to get to the same state as this one since I Squash & merged your PR

**Pull request merged**